prediction_market_agent/agents/mech_agent/deploy.py

```diff
@@ -25,10 +27,10 @@ def prediction_fn(self) -> t.Callable[[str, MechTool], OutcomePrediction]:
         return mech_request_local if self.local else mech_request
 
     def pick_markets(self, markets: t.Sequence[AgentMarket]) -> t.Sequence[AgentMarket]:
-        # We simply pick 5 random markets to bet on
+        # We randomly pick markets to bet on
         markets = list(markets)
         random.shuffle(markets)
-        return markets
+        return markets[: self.max_markets_per_run]
```

Comment:
oops, subset of markets weren't being picket!

Reply:
Why doesn't this create a conflict with my PR 🤔 https://github.com/gnosis/prediction-market-agent/pull/113/files#diff-dcd02995797aba7b69369a798982476199cff14bbbebdbdce844e9fddc3fe047R31

It's nicer like this just wondering why Github isn't complaining

Reply:
Ah I missed that comment! The base branch for this PR wasn't main. Now it's changed to main, there is a conflict 😄